### PR TITLE
Implement db helper method to retrieve available listings by market and date range

### PR DESCRIPTION
**Pull request by christine-w**

### CODE REVIEW

File: inventoryLocalStore/index.js [hunk 1]
```diff
@@ -1,10 +1,11 @@
 const { Pool } = require('pg');
 
+const getLengthOfStay = (start, end) =>
+  ((new Date(end) - new Date(start)) / 86400000) + 1;
+
```
Comment (tylertruong) [inline inventoryLocalStore/index.js]: Might be worth a code explanation for later. Hard to see that 86400000 = 1 day.

File: inventoryLocalStore/index.js [hunk 2]
```diff
@@ -14,6 +15,29 @@ class Inventory {
       .then(result => result.rows)
       .catch(console.error);
   }
+
+  getAvailableListings(market, start, end, limit) {
+    const queryLimit = limit ? ` FETCH FIRST ${limit * getLengthOfStay(start, end)} ROWS ONLY` : '';
+    const queryString = `
+      SELECT DISTINCT *
+      FROM listings l
```
Comment (tylertruong) [inline inventoryLocalStore/index.js]: Clarify what l is or make it more descriptive.

**Review by tylertruong — COMMENTED** [review]
This is great! 👍 

I need to review database queries so did not test your query as extensively as I should. Perhaps @tiffaphm or @ryanakiyama can give a second opinion.

File: inventoryLocalStore/index.js (hunk 1 above)
Comment (christine-w) [reply inventoryLocalStore/index.js]: Noted. I will update PR.

File: inventoryLocalStore/index.js (hunk 2 above)
Comment (christine-w) [reply inventoryLocalStore/index.js]: OK, I'll add the "AS" syntax to make the aliasing more clear.

**Pull request merged**